docs/pages/bare/overview.mdx

```diff
@@ -25,52 +26,183 @@ If you've ever written a native module for React Native, you'll be surprised how
 
 There's so much more to explore, and the links below will help you to explore the options available to you.
 
-## Next Steps
+## Incremental adoption steps
+
+There are four steps of incremental adoption to help you get started.
+
+### Prerequisites
+
+Required to use Expo tools:
 
 <BoxLink
   title="Install Expo modules"
   Icon={Rocket02Icon}
   description={
     <>
-      If you have already initialized a React Native app without <CODE>create-expo-app</CODE>, learn
-      how to install the Expo module library.
+      To unlock Expo capabilities, you need to install <CODE>expo</CODE> package in your project in
+      your existing React Native project. This guide provides both automated and manual installation
+      steps.
     </>
   }
   href="/bare/installing-expo-modules"
 />
+
 <BoxLink
-  title="Use Expo SDK"
+  title="Use Expo CLI"
   Icon={BookOpen02Icon}
+  description={
+    <>
+      Migration to Expo CLI provides many benefits over <CODE>@react-native-community/cli</CODE>.
```

Comment:
```suggestion
      Migration to Expo CLI provides many benefits over <CODE>@react-native-community/cli</CODE>.
```

Reply:
Better wording isn't coming to me at the moment, but something that emphasizes full compatibility with all Expo features and that it's more-or-less a drop-in replacement for both seems about right.

Reply:
Updated it to:

```md
Migration to Expo CLI is a drop-in replacement for <CODE>@react-native-community/cli</CODE>.
      It provides full compatibility with all Expo tools and services. This guide explains the
      benefits and provides compilation commands to start your development server after installing
      <CODE>expo</CODE> package.
```